config/environments/production.rb

```diff
@@ -78,6 +78,6 @@
   config.active_record.dump_schema_after_migration = false
 
   #Devise default urls for action mailer options
-  config.action_mailer.default_url_options = { host: 'intense-escarpment-81403.herokuapp.com' }
+  config.action_mailer.default_url_options = { host: 'intense-escarpment-81403.herokuapp.com', port: 3000 }
```

Comment:
Nice to see this site deployed to Heroku!  You may need to run some commands to remotely create or migrate the db.  I ran into some errors when I tried to login.  This is low priority though, I can show you in our next meeting how to do this...